src/features/unified-share-modal/utils/convertData.js

```diff
@@ -160,7 +181,8 @@ export const convertItemResponse = (itemAPIData: ContentSharingItemAPIResponse):
 
         sharedLink = {
             accessLevel,
-            allowedAccessLevels: ALLOWED_ACCESS_LEVELS,
+            accessLevelsDisabledReason: shared_link_access_levels_disabled_reasons || {},
```

Comment:
Minor: add default value `{}` while destructuring `shared_link_access_levels_disabled_reasons ` from `itemAPIData`

Reply:
I tried to do this initially, but unfortunately, it didn't work. The `fillMissingProperties` function (called after a successful response from the API) sets all missing properties to `null`, and destructuring with default values only works when the property is `undefined`.